myproject/supermarket_manager/constants.py

```diff
@@ -52,3 +52,9 @@
 
 # Message for required email or password
 REQUIRED_LOGIN = "Both email and password are required"
+
+# Message for change password
+CHANGE_PASS = "Password changed successfully"
```

Comment:
Should have been named more clearly, for example: CHANGED_PASSWORD

Reply:
Done